front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -1,29 +1,47 @@
-import { Fragment, useCallback, useEffect, useRef, useState } from 'react';
+import { Fragment, useCallback, useEffect, useMemo, useRef, useState } from 'react';
 
-import { X, ChevronDown, Question } from '@osrd-project/ui-icons';
+import { X, ChevronDown, Question, ChevronUp } from '@osrd-project/ui-icons';
 import cx from 'classnames';
 import { useTranslation } from 'react-i18next';
 
 import useAuth from 'utils/hooks/useAuth';
+import type { InfraWithState, ScenarioResponse } from 'common/api/generatedEditoastApi';
+import InfraLoadingState from 'applications/operationalStudies/components/Scenario/InfraLoadingState';
+import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
+import AddAndEditScenarioModal from 'modules/scenario/components/AddOrEditScenarioModal';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 
 type ScenarioHeaderProps = {
-  scenarioName: string;
+  scenario: ScenarioResponse;
+  infra: InfraWithState;
 };
 
-const ScenarioHeader = ({ scenarioName }: ScenarioHeaderProps) => {
+const ScenarioHeader = ({ scenario, infra }: ScenarioHeaderProps) => {
   const { username } = useAuth();
-  const { t } = useTranslation('common');
+  const { openModal } = useModal();
+
+  const { t } = useTranslation(['translation', 'common', 'operational-studies']);
```

Comment:
You have more operational-studies translations in this file, so it would be better to give this simple arg:
```suggestion
  const { t } = useTranslation('operational-studies');
```

Then you can remove all the `t(operational-studies:...)` => `t(...)`